cql/ManageRareAbnormality.cql

```diff
@@ -613,6 +613,9 @@ define RecommendationForExceptionsToColposcopyThreshold:
 define MostRecentBiopsyReportWasWithinPastYear:
   Collate.MostRecentBiopsyReport.date >= Now() - 1 year
 
+define MostRecentBiopsyReportWasWithinPastThreeYears:
```

Comment:
This appears to be unused throughout the CQL logic.

Reply:
This function is used in the updated I.2.2 which is part of[ PR 120](https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/120).